grafana/grafana.ini

```diff
@@ -231,21 +231,21 @@ check_for_updates = true
 [log]
 # Either "console", "file", "syslog". Default is console and  file
 # Use space to separate multiple modes, e.g. "console file"
-;mode = console, file
+;mode = console
 
 # Either "trace", "debug", "info", "warn", "error", "critical", default is "info"
 ;level = info
 
 # For "console" mode only
 [log.console]
-;level =
+;level = info
```

Comment:
All grafana options are configurable via environment variables by default. I believe this could be overridden with `LOG_CONSOLE_LEVEL=warn` set in docker-compose.yml.